substrate/client/rpc-servers/src/lib.rs

```diff
@@ -163,20 +159,38 @@ where
 		methods: build_rpc_api(rpc_api).into(),
 		service_builder: builder.to_service_builder(),
 		metrics,
-		tokio_handle,
-		stop_handle: stop_handle.clone(),
+		tokio_handle: tokio_handle.clone(),
+		stop_handle,
+		rate_limit_whitelisted_ips: Arc::new(rate_limit_whitelisted_ips),
 	};
 
-	let make_service = make_service_fn(move |addr: &AddrStream| {
```

Comment:
hyper::service::make_service_fn is removed and using to low-level API should easy to add support for ipv6 sockets
but I'll fix it another PR.